Respawn/DbAdapter.cs

```diff
@@ -39,6 +39,12 @@ public string BuildTableCommandText(Checkpoint checkpoint)
 
                     commandText += " AND t.name NOT IN (" + args + ")";
                 }
+                if (checkpoint.TablesToInclude.Any())

```

Comment:
This is only SQL Server, how would the other adapters need to change to support this behavior?

Reply:
I implemented and tested the other adapters.